demos/connector/neo4j/undirected-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -0,0 +1,1117 @@
+{
```

Comment:
<p>I think this looks a bit simpler with something like this:</p><pre class="ql-syntax" spellcheck="false"># flatten out the features
features_list = node_data["attr"].apply(pd.Series)["features"].values.tolist()

# create dataframe
node_features = pd.DataFrame(features_list, index=node_data["id"])
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>